employee-rostering-frontend/tslint.json

```diff
@@ -0,0 +1,50 @@
+{
+  "extends": [
+    "tslint:recommended",
+    "tslint-react",
+    "tslint-config-airbnb"
+  ],
+  "linterOptions": {
+    "exclude": [
+      "build/**",
+      "config/**/*.js",
+      "node_modules/**/*.ts",
+      "coverage/lcov-report/*.js"
+    ]
+  },
+  "rules": {
+    "file-header": [
+      true,
+      "^\n \\* Copyright \\d{4} Red Hat, Inc\\. and/or its affiliates\\.\n \\*\n \\* Licensed under the Apache License, Version 2\\.0 \\(the \"License\"\\);\n \\* you may not use this file except in compliance with the License\\.\n \\* You may obtain a copy of the License at\n \\*\n \\*     http://www\\.apache\\.org/licenses/LICENSE-2\\.0\n \\*\n \\* Unless required by applicable law or agreed to in writing, software\n \\* distributed under the License is distributed on an \"AS IS\" BASIS,\n \\* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied\\.\n \\* See the License for the specific language governing permissions and\n \\* limitations under the License\\.\n $",
+      "Copyright 2019 Red Hat, Inc. and/or its affiliates.\n\nLicensed under the Apache License, Version 2.0 (the \"License\");\nyou may not use this file except in compliance with the License.\nYou may obtain a copy of the License at\n\n    http://www.apache.org/licenses/LICENSE-2.0\n\nUnless required by applicable law or agreed to in writing, software\ndistributed under the License is distributed on an \"AS IS\" BASIS,\nWITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.\nSee the License for the specific language governing permissions and\nlimitations under the License."
+    ],
+    "import-name": false,
+    "interface-name": [true, "never-prefix"],
+    "function-name": [
+      {
+        "function-regex": "^[a-zA-Z][\\w\\d]+$"
+      }
+    ],
+    // In some edge cases: https://github.com/wmonk/create-react-app-typescript/issues/370#issuecomment-415299545
+    // WHY CAN BE DANGEROUS: https://maarten.mulders.it/blog/2017/07/no-bind-or-arrow-in-jsx-props-why-how.html
+    "jsx-no-lambda": false,
+    "jsx-no-multiline-js": false,
+    "max-line-length": [true, 120],
+    "member-access": [
+      true,
+      "no-public"
+    ],
+    "no-console": {
+      "severity": "warn"
+    },
+    "object-literal-sort-keys": false,
+    "object-shorthand-properties-first": false,
+    // This rule is here for now because of https://github.com/palantir/tslint-react/issues/120
+    "variable-name": [
+      true,
+      "ban-keywords",
+      "check-format",
+      "allow-pascal-case"
+    ]
+  }
+}
```

Comment:
This file is a leftover from the period of using `react-scripts-ts`. Please replace with `.estlintrc.json` from VRP's master.